### ISSUE TITLE
More data about exclusions.

### ISSUE THREAD

**Issue by davorg** [davorg/succession#45]

**Comment by ras52** [comment]
### The Earl of St Andrews' children

George Windsor, Earl of St Andrews, the oldest son of the current Duke of Kent married a Catholic on 1988-01-09, which you correctly show.  However it appears that only his oldest two children are considered Catholic for the purpose of the succession.  The Bill of Rights and Act of Settlement are not clear exactly what constitutes a Catholic, but reading a bit about it, it seems being confirmed is the act which excludes you from the succession.  E.g. see the third paragraph under the heading "Edward Windsor, Lord Downpatrick", [here](https://en.wikipedia.org/wiki/George_Windsor,_Earl_of_St_Andrews#Edward_Windsor,_Lord_Downpatrick).  If this is correct, and if the information on Wikipedia is up to date, then one of St Andrews' children remains in the line of succession:
* <s>Edward Edmund Maximilian George Windsor, Lord Downpatrick, b 1988-12-02.</s>  (Confirmed Catholic.)
* <s>Lady Marina Charlotte Alexandra Katharine Helen Windsor, b 1992-09-30.</s>  (Confirmed Catholic.)
* Lady Amelia Sophia Theodora Mary Margaret Windsor, b 1995-08-24.

This poses a problem in that we really need the dates of confirmation.   A bit of Googling suggests Downpatrick's confirmation was in May/June 2003, and Lady Marina Windsor was confirmed in [early 2008](https://www.theguardian.com/theobserver/2008/may/25/1), but the exact dates seem hard to find.

**Comment by ras52** [comment]
### Lord Nicholas Windsor's family

Lord Nicholas Windsor, the current Duke of Kent's younger son, [converted to Catholicism in 2001](http://www.telegraph.co.uk/news/uknews/theroyalfamily/9076668/The-Queens-cousin-says-We-are-prejudice-victims.html), but again no exact date.  His wife (whom he married 2006-11-04) is a Catholic and his children were baptised as Catholics.   
* Albert Louis Philip Edward Windsor, b 2007-09-22.
* Leopold Ernest Augustus Guelph Windsor, b 2009-09-08.
* Louis Arthur Nicholas Felix Windsor, b 27 May 2014.

From their ages, I would expect any of these sons to have been confirmed yet, so presumably they remain in the line of succession for now.

**Comment by davorg** [comment]
Yes, I thought this was going to get difficult. I wonder if the Catholic church maintains a public list of confirmations.

Thanks, as always, for your efforts here.

**Comment by ras52** [comment]
Note that at present, none of the children of the Duke of Kent's two sons (the Earl of St Andrews and Lord Nicholas Windsor) seem to listed on the site, whether excluded or not.

**Comment by davorg** [comment]
I emailed Burke's Peerage (in Jan 2018) and they recommended speaking to the Lord Chamberlain's office.
